src/wizard/obtain_source.jl

```diff
@@ -352,6 +352,30 @@ function get_name_and_version(state::WizardState)
     end
 end
 
+@enum Compilers C=1 Go Rust
```

Comment:
I'm not really sure what this `@enum` is adding?  Why not just create a list of `["C/C++/Fortran", "Go", "Rust"]` and use the indices directly?

Reply:
I'd have two vectors here: vectors of strings with the descriptions, vector of symbols, which is a bit tricky to keep in sync.  I think in this way I don't have to care about the order of the elements of the vectors and it's easier to add new compilers or change the description and the name of a compiler.